include/vctr/Containers/VctrBase.h

```diff
@@ -366,6 +366,24 @@ class VctrBase : private Config,
     /** Subtracts a constant from this in place */
     void operator-= (value_type c);
 
+    //==============================================================================
+    // Math reduction operations
+    //==============================================================================
+    /** Returns the minimal value of all elements */
+    ElementType min() const requires std::totally_ordered<ElementType>;
+
+    /** Returns the minimal absolute value of all elements */
+    ElementType minAbs() const requires std::totally_ordered<ElementType>;
+
+    /** Returns the maximum value of all elements */
+    ElementType max() const requires std::totally_ordered<ElementType>;
+
+    /** Returns the maximum absolute value of all elements */
+    ElementType maxAbs() const requires std::totally_ordered<ElementType>;
+
+    /** Returns the sum of all elements */
+    ElementType sum() const requires has::operatorPlusEquals<ElementType>;
+
 protected:
```

Comment:
This header uses //============== separators in some places. For consistency and because this is a very long header, I suggest also putting them after access specifiers although this line is not part of the PR!